OasisDEX/oasis-borrow
# Constant multiple borrow validation

# [Constant multiple - borrow validation](https://app.shortcut.com/oazo-apps/story/5627/validation-cm-sell-buy-trigger-validation-in-borrow)

# [Constant multiple - multiple validation](https://app.shortcut.com/oazo-apps/story/5628/validation-cm-sell-buy-trigger-validation-in-multiply)
<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added validation to borrow and multiply vaults regarding constant multiple buy and sell triggers
  
## How to test 🧪
  <Please explain how to test your changes>

- use constant multiple feature toggle
- add CM trigger to your vault which wont be triggered immediately
- in overview try to deposit or withdraw big amount of collateral, error should appear in both cases with slightly different message and button should be disabled
